app/packages/acter_trigger_auto_complete/lib/src/acter_text_controller.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+
+/// Extended [TextEditingController] which takes up trigger styles as optional.
+/// Styles the trigger inputs based on map for mentions, hashtags or emojis etc.
+/// If provided empty, will
```

Comment:
will ...?

Reply:
Sorry, missed the comment here. Needs correction! 👍🏻